Base/Dockerfile

```diff
@@ -18,6 +18,7 @@ RUN apt-get update -qqy \
     sudo \
     unzip \
     wget \
+    bzip2 \
```

Comment:
The list of packages to install is sorted alphabetically to minimize the risk of merge conflicts.